Sources/SwiftGRPC/Core/CompletionQueue.swift

```diff
@@ -44,12 +44,12 @@ enum CompletionType {
 struct CompletionQueueEvent {
   let type: CompletionType
   let success: Int32
-  let tag: Int64
+  let tag: Int
 
   init(_ event: grpc_event) {
     type = CompletionType.completionType(event.type)
     success = event.success
-    tag = cgrpc_event_tag(event)
+    tag = Int(bitPattern:cgrpc_event_tag(event))
```

Comment:
nit: Looks like elsewhere in the codebase there's normally a space between the `:` and the argument - i.e., `Int(bitPattern: cgrpc_event_tag(event))`. Same below in `Handler.swift` 🤷‍♂️